.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just FYI for the future -- since these are your own personal preferences, it's best practice to add this file to your .gitignore